app/scripts/controllers/accounting/provisioning/ViewAllProvisoningEntriesController.js

```diff
@@ -24,6 +25,10 @@
                 var params = {};
                 params.offset = offset;
                 params.limit = limit;
+                if (scope.filterDate) {
+                    params.filterDate = dateFilter(scope.filterDate, scope.df);
```

Comment:
I dont see "filterDate" support in ProvisioningEntriesApiResource (i.e /provisioningentries/:entryId) would you please check on this.